mysql-test/suite/versioning/r/alter.result

```diff
@@ -125,7 +125,10 @@ with system versioning;
 select * from t for system_time all;
 a
 alter table t drop column sys_trx_start;
+ERROR HY000: Wrong parameters for `t`: missing 'DROP COLUMN sys_trx_end'
 alter table t drop column sys_trx_end;
+ERROR HY000: Wrong parameters for `t`: missing 'DROP COLUMN sys_trx_start'
+alter table t drop column sys_trx_start, drop column sys_trx_end;
```

Comment:
Dropping of system columns must be prohibited while SYSTEM VERSIONING is in effect. When SYSTEM VERSIONING is off, system columns become plain columns and can be dropped one-by-one. We don't have such option to turn off SYSTEM VERSIONING yet.

Reply:
Well, maybe you're right. But what to do with MDEV-14798? Suspend for now? Or commit other parts like invisibility requirement?

Reply:
For this specific test case prohibit dropping of system columns.